extensions-core/kafka-indexing-service/src/test/java/io/druid/indexing/kafka/KafkaIndexTaskClientTest.java

```diff
@@ -181,7 +180,7 @@ public void testInternalServerError()
     verifyAll();
   }
 
-  @Test(expected = IAE.class)
+  @Test(expected = RuntimeException.class)
```

Comment:
I think `RuntimeException` is too broad, and this check might be false positive. Would you improve this test by adding `ExpectedException` and checking the cause and the error message of the `RuntimeException`?

Reply:
Changed this and other tests here to use ExpectedException and also check the exception messages